### PR TITLE
Created welcome.md

### PR DESCRIPTION
**Pull request by mary-grace**
first draft at a welcome DM for new users

### CODE REVIEW

**Comment by colestrode** [comment]
Love it 😄

File: welcome.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+:wave: Hi there, and welcome to the SparkPost community! I'm Sparky, your friendly Slackbot, and I'm here to give you a quick tour so you feel a little more at home.
```
Comment (colestrode) [inline welcome.md:1]: let's add a space between "Slack" and "bot" -> "I'm Sparky, your friendly Slack bot,...."

Slackbot is the built in bot in slack :)

File: welcome.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Start off in the #community channel. Feel free to introduce yourself... we're a friendly crew :) Tell us who you are and where you're from, or just wave hello. We'd love to get to know you!
+
+If you've got a general question about getting started with SparkPost, head over to #support. Someone there will be able to help you, or at least point you in the right direction.
```
Comment (colestrode) [inline welcome.md:5]: channel got renamed to #community-support

File: welcome.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Know exactly what topic you need help with? Take a look at the list of channels -- you just may find the one you're looking for! Whether you'll find a droid there is another question entirely.
+
+If you can't find the droid, er, topic you're looking for, feel free to create a new channel, and then alert a few people to it. Chances are, there are other people interested in that same topic who would be more than willing to join you. Keep in mind that public channels are always preferred to private. For other witty suggestions on how to best use Slack channels, see [Emily Post's Unofficial Guide to Slack Etiquette](https://github.com/SparkPost/slack-etiquette).
```
Comment (colestrode) [inline welcome.md:9]: maybe we don't need "witty" in "For other witty suggestions"?

**Comment by colestrode** [comment]
Sorry I didn't see this before! I added a few comments, but this is great :)

Do you think we should have a sentence about this channel not being "official" support?

**Comment by mary-grace** [comment]
yep - that makes sense. I'll add something in there about it.

**Pull request merged**